src/main/java/com/manardenza/service/ReservationService.java

```diff
@@ -0,0 +1,57 @@
+package com.manardenza.service;
+
+import com.manardenza.dao.ReservationDaoImpl;
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Reservation;
+import com.manardenza.entity.Room;
+import com.manardenza.login.CurrentUser;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public final class ReservationService {
+
+    private static ReservationService instance;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "instance" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)